src/Microsoft.DocAsCode.Build.TableOfContents/Utility.cs

```diff
@@ -37,18 +37,36 @@ public static bool IsSupportedRelativeHref(HrefType hrefType)
                 || hrefType == HrefType.MarkdownTocFile;
         }
 
-        public static TocViewModel LoadSingleToc(string file)
+        public static TocItemViewModel LoadSingleToc(string file)
```

Comment:
we will refactor in seprate review

---
In reply to: [120289039](https://github.com/dotnet/docfx/pull/1731#discussion_r120289039) [](ancestors = 120289039)